### PR TITLE
[video.js] - Added currentBreakpoint and currentBreakpointClass methods

### PR DESCRIPTION
**Pull request by try2beth3b3st**
https://docs.videojs.com/player#currentBreakpoint
https://docs.videojs.com/player#currentBreakpointClass

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
Please review CI log and also please add the actual code usage in tests (as a way to test a change):
https://github.com/DefinitelyTyped/DefinitelyTyped#test-editing-an-existing-package

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## video.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/video.js@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'video.js'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. hooks_
1. hooks
1. hook
1. hookOnce

<details>
<summary>as well as these 52 other properties...</summary>
<p>removeHook, VERSION, options, getPlayers, getPlayer, getAllPlayers, players, getComponent, registerComponent, getTech, registerTech, use, middleware, browser, TOUCH_ENABLED, extend, mergeOptions, bind, registerPlugin, deregisterPlugin, plugin, getPlugins, getPlugin, getPluginVersion, addLanguage, log, createLogger, createTimeRange, createTimeRanges, formatTime, setFormatTime, resetFormatTime, parseUrl, isCrossOrigin, EventTarget, on, one, off, trigger, xhr, TextTrack, AudioTrack, VideoTrack, computedStyle, dom, url, mediaSources, MediaSource, URL, HlsHandler, HlsSourceHandler, Hls</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dd74e4097473e8083bc5f066f8315a96da2f7136
</p>

**Comment by try2beth3b3st** [comment]
> Please review CI log and also please add the actual code usage in tests (as a way to test a change): https://github.com/DefinitelyTyped/DefinitelyTyped#test-editing-an-existing-package

done

File: types/video.js/video.js-tests.ts [hunk 1]
```diff
@@ -159,6 +159,12 @@ videojs('example_video_1', playerOptions).ready(function playerReady() {
 
     const readyState: videojs.ReadyState = this.readyState();
 
+    // $ExpectType string
+    const currentBreakPoint: string = this.currentBreakpoint();
```
Comment (peterblazejewicz) [inline types/video.js/video.js-tests.ts:163]:
```suggestion
    const currentBreakPoint = this.currentBreakpoint();
```
(as per assertion, type is infered)

File: types/video.js/video.js-tests.ts [hunk 2]
```diff
@@ -159,6 +159,12 @@ videojs('example_video_1', playerOptions).ready(function playerReady() {
...
+
+    // $ExpectType string
+    const currentBreakpointClass: string = this.currentBreakpointClass();
```
Comment (peterblazejewicz) [inline types/video.js/video.js-tests.ts:166]:
```suggestion
    const currentBreakpointClass = this.currentBreakpointClass();
```
(ditto)

**Comment by try2beth3b3st** [comment]
@peterblazejewicz excuse me, can you review last fix?)

**Review by peterblazejewicz — APPROVED** [review]

**Comment by try2beth3b3st** [comment]
Ready to merge

**Pull request merged**